app/javascript/packages/components/block-link.tsx

```diff
@@ -1,18 +1,13 @@
 import Link, { LinkProps } from './link';
 import BlockLinkArrow from './block-link-arrow';
 
-export interface BlockLinkProps extends LinkProps {
-  /**
-   * Link destination.
-   */
-  href: string;
-}
```

Comment:
I think removing this will make usages more prone to stumble across accessibility issues (tabbing and activating the link) unless we provide an alternative. I did run into problems with the  `javascript:void(0);`, so it'd probably have to be a different solution.

Reply:
Disregard, I see that this is already required by `LinkProps`.